### ISSUE TITLE
UPDATE: Data no longer available from GitHub repository, use openexchangerates.org instead

### ISSUE THREAD

**Issue by wjcrowcroft** [currencybot/open-exchange-rates#35]
Had an email from GitHub today, explaining that some people have been _hammering_ the files on the repository and putting a strain on resources, which is negatively affecting performance for other users. 

**[Open Exchange Rates](http://openexchangerates.org)** began life on GitHub because it was the quickest and dirtiest way to get the service up and running. Without it, the project wouldn't exist - so we owe a great deal to them - but they've politely asked me to **remove the data from the repository**.
### More info:

**[Read the full blog post here!](http://www.josscrowcroft.com/2012/projects/open-exchange-rates-update-the-partys-not-over-it-just-got-a-little-too-noisy/)**
### Long story short:
- If you're using GitHub to get the data, you'll need to start using **[openexchangerates.org](http://openexchangerates.org)** - it will soon support HTTPS and extended queries.
- The site is re-launching at the end of next week with a basic signup and new improved featured and documentation
- To prevent the server from being crushed under the weight of all these requests, you'll need an API key (App ID) starting from next month.
- App IDs are free (or "pay what you like") for personal/small scale use, and dirt-cheap for business/enterprise use.

**Comment by wjcrowcroft** [comment]
### Update:

Data has now been removed from the GitHub repository. I also removed the `code/` directory since those were deprecated scripts no longer in use.

The data are all still available at **[openexchangerates.org](http://openexchangerates.org)** - it will always be free to get the non-commercial data, but to keep the project alive, I'm exploring a "free or _pay-what-you-like_" model, and implementing the advanced features that many of you have requested.

From next week, you'll be able to sign up for a free App ID - and from August, these will be required to access the API. You can **[stay updated on Twitter](https://twitter.com/josscrowcroft)**.

This repository will be filled with code examples for working with the API.

**Comment by wjcrowcroft** [comment]
### Update 2012-07-15:

App ID signup is now open at **[openexchangerates.org/signup](https://openexchangerates.org/signup)**.
